apps/src/javalab/PreviewPaneHeader.jsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
 import PaneHeader, {
   PaneSection,
   PaneButton
 } from '@cdo/apps/templates/PaneHeader';
 import CollapserIcon from '@cdo/apps/templates/CollapserIcon';
 
-export default function PreviewPaneHeader({isCollapsed, isFullscreen}) {
+export default function PreviewPaneHeader({
+  isCollapsed,
+  isFullscreen,
+  showAssetManagerButton = false,
+  showPreviewTitle = true
```

Comment:
nit: set these as default props below the propTypes definition, e.g.:

```javascript
PreviewPaneHeader.propTypes = {};

PreviewPaneHeader.defaultProps = {
  showAssetManagerButton: false,
  showPreviewTitle: true
};
```

Reply:
👍 is that our preferred style? I wasn't sure and was googling and found this when I was deciding which to do:

https://stackoverflow.com/questions/47774695/react-functional-component-default-props-vs-default-parameters

Reply:
ooo, interesting -- TIL! i mostly see us using `defaultProps` for functional components, but if that will be deprecated, let's keep this as-is and we'll start transitioning to default params